### PR TITLE
continuos-integration

### PR DESCRIPTION
**Pull request by stefymojica**

### CODE REVIEW

File: .nvmrc [hunk 1]
```diff
@@ -0,0 +1 @@
+v18.3.0
```
Comment (kliver98) [inline .nvmrc]: Please use a stable version (an LTS version, any 16.x should work)

File: .github/workflows/main.yml [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Setup
+        uses: actions/setup-node@v2
+        with:
+          node-version: 16
+          cache: "yarn"
```
Comment (danielgalvis98) [inline .github/workflows/main.yml:16]: I'm not too familiar with yarm, but I think that a similar behavior to `npm ci` would be something like `yarn install --immutable`

File: .nvmrc (hunk 1 above)
Comment (stefymojica) [reply .nvmrc]: Fixed, thanks

File: .github/workflows/main.yml (hunk 2 above)
Comment (stefymojica) [reply .github/workflows/main.yml:16]: I checked the documentation (https://github.com/actions/setup-node#caching-global-packages-data) that indicates it is the package manager and not a command.

Comment (danielgalvis98) [reply .github/workflows/main.yml:16]: Sorry, I put the comment on the wrong place. You are righ, here it is correct

File: .github/workflows/main.yml [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+
+      - name: Install
+        run: yarn
```
Comment (danielgalvis98) [inline .github/workflows/main.yml]: Here is the place where I refered that I think that a similar behavior to` npm ci` would be something like `yarn install --immutable`. My mistake

File: .github/workflows/main.yml [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
```
Comment (danielgalvis98) [inline .github/workflows/main.yml]: I'm not too familiar with yarm, but I think that a similar behavior to `npm ci` would be something like `yarn install --immutable` (This was the right place)

Comment (stefymojica) [reply .github/workflows/main.yml]: Yes,  yarn install --immutable is when using yarn 2, In my case I  use version 1 and it is  yarn install --frozen-lockfile

**Review by danielgalvis98 — APPROVED** [review]

**Pull request merged**